docs/Installation.md

```diff
@@ -26,80 +25,89 @@ To set up a user with publish and subscribe permission, open the IE Databus Conf
 - Click "Add Topic" and enter a topic name, e.g `CloudConnector/toAWS` and click save. This topic must start with the name configured in the previous step.
-- Click on Add Route
-- enter a name for the route
 
-The last part is to configure the cloud connector client
+![iot gateway adapters](./graphics/iot-gateway-adapters.png)
+
+- Click on Add Route 
+- Enter a name for the route `toAWS`
+
+![iot gateway route](./graphics/iot-gateway-route.png)
+
+The last part is to configure the cloud clients
 
 - Click on "Add Client"
-- Enter a Name e.g `awsClient` and choose `AWS` as type and click "save"
-- select the certificate and private key you downloaded when creating the aws thing
-- click on the edit symbol net to the client
-- enter a Client ID e.g `ccID`. This will be the ID used by the connector to publish data to the MQTT broker of the aws IoT device.
-- enter the hostname of the aws endpoint. You can find the address at the "Settings" section of the aws IoT console. It looks similar to this: `abcdefghi12345.iot.us-east.amazonaws.com`
-- enter `8883` as Port Number
-- enter Proxy settings if needed
-- Now select the route, check the box next to the bus adaptor topic `CloudConnector/toAWS` and check the box next to Cloud Connector Clients `awsClient`
-- click "save"
-
-![cc configurator setup](./graphics/iot-gateway-cc-configurator.gif)
+- Enter a Name e.g `ccID` and choose `AWS` as type and click "save"
+- Select the certificate and private key you downloaded when creating the aws thing
+- Click on the edit symbol net to the client
+- Enter a Client ID e.g `ccID`. This will be the ID used by the connector to publish data to the MQTT broker of the aws IoT device.
+- Enter the hostname of the aws endpoint. You can find the address at the "Settings" section of the aws IoT console. It looks similar to this: `abcdefghi12345.iot.us-east.amazonaws.com`
+- Enter `8883` as Port Number
+- Enter Proxy settings if needed
+
+![iot gateway client configuration](./graphics/iot-gateway-client-configuration.png)
+![iot gateway client configuration advance](./graphics/iot-gateway-client-configuration-advance.png)
+
+- Now select the route, check the box next to the bus adaptor topic `CloudConnector/toAWS` and check the box next to Cloud Connector Clients `ccID`
+- Click "save" and "Deploy"
+
+![iot gateway save deploy](./graphics/iot-gateway-save-deploy.png)
 
 Now every data published to the bus adaptor topic of the IE Databus will get published to the aws cloud.
 E.g. data published to the topic `CloudConnectot/toAWS` in the IE Databus will get published to the topic `CloudConnector/` of the aws IoT thing. The publish topic to which the IE Cloud Connector will publish the data in the aws cloud is set in the "Advanced" Tab of the IE Cloud Connector Client settings, the default value is `CloudConnector/`
```

Comment:
Please check this section
In Screenshot Cloud Client configuration you send data to CloudConnector/test

When I read this here it sound like it is send to CloudConnector/ please update this

Additional
aws => AWS